src/radio/radio-group.component.ts

```diff
@@ -152,7 +152,17 @@ export class RadioGroup implements AfterContentInit, AfterViewInit, ControlValue
 	/**
 	 * Set to true to disable the whole radio group
 	 */
-	@Input() disabled = false;
+	@Input()
+	set disabled(disabled: boolean) {
+		this._disabled = disabled;
+		this.updateRadios();
+	}
+	/**
+	 * Returns the disabled value for the `RadioGroup`.
+	 */
+	get disabled(): boolean {
+		return this._disabled;
```

Comment:
Where is `_disabled` defined and initialized?

Reply:
line 198...it was already there but never used